### PR TITLE
[WIP] Add local test suite to dinersclub

### PR DESCRIPTION
**Pull request by calufa**
To try it out, execute in your terminal, `./test.sh dinersclub`

TODO:

- [x] Add infrastructure for local testing
- [x] Use config.go to store system-level variables and constants.
- [x] Replace "ADD" for "COPY" in Dockerfile
- [ ] Make sure all tests are passing

### CODE REVIEW

**Comment by nandanrao** [comment]
LGTM! Although I don't know why the tests aren't passing... ?

File: dinersclub/dinersclub_test.go [hunk 1]
```diff
@@ -86,22 +124,27 @@ func TestDinersClubErrorsOnMalformedJSONMessages(t *testing.T) {
...
-          "pageid": "page",
-          "timestamp"1600558963867
-          "details": {"result": {"type": "foo", "success":true}}}`,
-	})
-
+	msgs := makeMessages([]string{`
+		{
+			"userid": "foo",
+			"pageid": "page",
+			"timestamp": 1600558963867,
```
Comment (nandanrao) [inline dinersclub/dinersclub_test.go:131]: This JSON is supposed to be malformed, as per the test, but you fixed it!

**Comment by calufa** [comment]
Ouch. Accidentally closed this ticket by doing a _force push_.

**Comment by nandanrao** [comment]
Ha, that's one way push a squashed feature branch!

**Pull request closed without merging**